### PR TITLE
update trait name to branded snooze naming

### PR DESCRIPTION
**Pull request by thomasjohnkane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,20 +50,28 @@ Snooze provides a trait for your model, similar to the standard `Notifiable` tra
...
+
+// Schedule for new years eve
+$user->notifyAt(new NewYearNotification, Carbon::parse('last day of this year'));
 ```
 
 #### Using the ScheduledNotification::create helper
 You can also use the `create` method on the `ScheduledNotification`. 
 ```php
-ScheduledNotification::create(
+ScheduledNotification::create([
```
Comment (atymic) [inline README.md]: there's no array syntax required here (it's a function)

**Comment by atymic** [comment]
Not sure why travis is failing - looks like the class is namespaced correctly 🤔

File: README.md (hunk 1 above)
Comment (thomasjohnkane) [reply README.md]: Oh duh, you had a bracket at the bottom, so I just added to top. Should have removed from bottom instead.

Comment (atymic) [reply README.md]: Oh, my bad.

Comment (atymic) [reply README.md]: Thanks!

**Pull request merged**